test/rulesets/Base/indexing.jl

```diff
@@ -68,3 +68,32 @@ end
     test_frule(setindex!, rand(3, 4), rand(), 1, 2)
     test_frule(setindex!, rand(3, 4), [1,10,100.0], :, 3)
 end
+
+@testset "eachslice" begin
+    # Testing eachrow not collect∘eachrow leads to errors, e.g.
+    # test_rrule: eachrow on Vector{Float64}: Error During Test at /Users/me/.julia/packages/ChainRulesTestUtils/8dFTY/src/testers.jl:195
+    #   Got exception outside of a @test
+    #   DimensionMismatch("second dimension of A, 6, does not match length of x, 5")
```

Comment:
Can we cross link the the issue on CRTU?
I think there might already be one?

Reply:
```suggestion
    #   DimensionMismatch("second dimension of A, 6, does not match length of x, 5")
    # Probably similar to https://github.com/JuliaDiff/ChainRulesTestUtils.jl/issues/234 (about Broadcasted not Generator)
```